src/emu/softlist.cpp

```diff
@@ -102,15 +102,12 @@ const device_type SOFTWARE_LIST = &device_creator<software_list_device>;
 //  software_part - constructor
 //-------------------------------------------------
 
-software_part::software_part(software_info &info, const char *name, const char *interface)
+software_part::software_part(software_info &info, const std::string &name, const std::string &interface)
```

Comment:
This is a good place to consume `std::string &&` and use `std::move` below to destructively consume them when initialising your members.